extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -71,6 +73,7 @@ type ProvisionFormData = {
   hostname: string;
   region: string;
   luisLocation: string;
+  appServiceOperatingSystem: string;
```

Comment:
nit: type the values as 'linux' | 'windows'

Reply:
I guess in theory there could be other values in the config.  I had this in the backend types, but then the compiler got angry with me because the value may not be in the config file.  I also hope the new extension replaces all this :)

Reply:
you can always "as MyType" it 😄